MFVNeutralinoFormats/interface/Event.h

```diff
@@ -27,7 +31,7 @@ struct MFVEvent {
     gen_valid = 0;
     npv = pv_ntracks = pv_ntracksloose = 0;
     gen_flavor_code = 0;
-    gen_weight = l1_htt = l1_myhtt = l1_myhttwbug = hlt_ht = hlt_caloht = npu = bsx = bsy = bsz = bsdxdz = bsdydz = bswidthx = bswidthy = pvx = pvy = pvz = pvcxx = pvcxy = pvcxz = pvcyy = pvcyz = pvczz = pv_score = metx = mety = metNoMux = metNoMuy = met_calo = 0;
+    gen_weight = l1_htt = l1_myhtt = l1_myhttwbug = hlt_ht = npu = bsx = bsy = bsz = bsdxdz = bsdydz = bswidthx = bswidthy = pvx = pvy = pvz = pvcxx = pvcxy = pvcxz = pvcyy = pvcyz = pvczz = pv_score = metx = mety = metNoMux = metNoMuy = met_calo = 0;
```

Comment:
Another one related to Shaun's work: will removing this break our ability to repeat the btag studies? @shaunhogan @pkotamnives

Reply:
Same question for all of the pass_, l1pass_, filtpass_, pass_filter, etc. in this file, and all of the hlt_* variables that have been removed.